### PR TITLE
Reintegrated Apollo Federation tests

### PR DESCRIPTION
**Pull request by danielreynolds1**
This PR changes some of the existing .txt files in the HotChocolate.ApolloFederation.Tests packages to code files and amends the tests such that they can be compiled and executed.

### CODE REVIEW

**Comment by michaelstaib** [comment]
Awesome... I had this for so long on my todo list :D 

Will have a look later today.

File: src/HotChocolate/ApolloFederation/global.json
Comment (glen-84) [inline src/HotChocolate/ApolloFederation/global.json:1]: This file shouldn't be necessary?

Comment (danielreynolds1) [reply src/HotChocolate/ApolloFederation/global.json:1]: I planned to rewrite the commit without this. Raised it as a draft to get support from Michael and maybe you as to how to get the @policy attribute to write correctly in the schema. Based on the tests, I don’t think it’s producing a valid federation schema at the moment

Comment (danielreynolds1) [reply src/HotChocolate/ApolloFederation/global.json:1]: Removed now!

**Pull request merged**